src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.form

```diff
@@ -199,6 +199,15 @@
           </component>
         </children>
       </grid>
+      <component id="bd9c9" class="javax.swing.JCheckBox" binding="overwriteExistingFilesCheckBox" default-binding="true">
```

Comment:
The `binding` should be `overwriteFilesField`

Reply:
Thanks. Will fix that. What is the use of that binding? Because it seems to work even though the binding is wrong.

Reply:
The .form files are just for the UI Designer. Since the UI Designer generates part of the code of the  corresponding .java file, we need to keep them in sync.

A trick to do that is to change name of members in the UI Designer and then, it takes care of everything :)